explore/src/main/scala/explore/targeteditor/AladinContainer.scala

```diff
@@ -68,15 +67,11 @@ object AladinContainer extends AladinCommon {
 
   // We need to dectect if the selected GS deserves a refresh, this could be if the
   // selected target changes or if e.g. the pos angle change for the same target
-  private given Reusability[Option[AgsAnalysis]] = Reusability.by(_ match {
-    case Some(AgsAnalysis.Usable(_, target, _, _, v)) => ((target.id, v)).some
-    // simulate vignetting for reusability it only matters if it changes
-    case Some(t)                                      =>
-      (t.target.id, NonEmptyList.of((Angle.Angle0, Area.MaxArea))).some
-    case _                                            => None
-  })
+  // private given Reusability[Option[AgsAnalysis.Usable]] =
+  private given Reusability[AgsAnalysis.Usable] =
+    Reusability.by(u => (u.target, u.posAngle))
```

Comment:
maybe just target.id?

Reply:
I was just reproducing what was already being used. I can switch to just target id if you think that is sufficient.

Reply:
I'll leave it because it shouldn't hurt anything. We can change it in another PR if we think it should be changed.

Reply:
ok